apps/communications/contacts/js/export/bt.js

```diff
@@ -15,7 +15,7 @@ var ContactsBTExport = function ContactsBTExport() {
   };
 
   var _hasDeterminativeProgress = function btex_hasDeterminativeProgress() {
-    return false;
+    return true;
```

Comment:
I've got no idea what this function is for, but I'm surprised that you can safely change the return value without changing anything else...